src/dynarmic/backend/x64/emit_x64_floating_point.cpp

```diff
@@ -116,9 +117,20 @@ void DenormalsAreZero(BlockOfCode& code, EmitContext& ctx, std::initializer_list
 
 template<size_t fsize>
 void ZeroIfNaN(BlockOfCode& code, Xbyak::Xmm xmm_value, Xbyak::Xmm xmm_scratch) {
-    code.xorps(xmm_scratch, xmm_scratch);
-    FCODE(cmpords)(xmm_scratch, xmm_value);  // true mask when ordered (i.e.: when not an NaN)
-    code.pand(xmm_value, xmm_scratch);
+    if (code.HasHostFeature(HostFeature::AVX512_OrthoFloat)) {
+        // Replace QNaN and SNaN with zero
+        constexpr u32 nan_to_zero = FixupLUT(
```

Comment:
```suggestion
        constexpr u32 nan_to_zero = FixupLUT(FpFixup::ZeroPos,
                                             FpFixup::ZeroPos);
```

Ditto for the occurrence below as well.